src/components/DatePicker/index.android.js

```diff
@@ -39,7 +38,7 @@ class DatePicker extends React.Component {
     }
 
     render() {
-        const dateAsText = this.props.value || this.props.defaultValue ? moment(this.props.value || this.props.defaultValue).format(CONST.DATE.MOMENT_FORMAT_STRING) : '';
+        const dateAsText = this.props.value || this.props.defaultValue ? format(new Date(this.props.value || this.props.defaultValue), CONST.DATE.FNS_FORMAT_STRING) : '';
```

Comment:
Can we create a variable like: `const date = this.props.value || this.props.defaultValue`, to re-use it?